README.md

```diff
@@ -71,13 +74,14 @@ sudo hcitool lescan
 A test run is as easy as:
 
 ```shell
-sudo ./gateway.py
+source .venv/bin/activate
+./gateway.py
```

Comment:
sudo seems to be required by bluepy lib, so if someone uses worker with them, sudo is needed.

Reply:
I see. It was working fine where I tested, but now that I have tried this on machines with other Linux releases I need to use sudo on at least one of them. Since I have no idea why (user groups appear to be identical) I have now re-added the `sudo` prefixes.